src/routes/(console)/project-[project]/overview/onboard.svelte

```diff
@@ -1,245 +1,548 @@
-                                        style="font-size: var(--icon-size-large);"
-                                        aria-hidden="true" />
-                                </span>
-                            </button>
-                        </li>
-                        <li class="grid-box-item">
-                            <a
-                                href={`${base}/project-${projectId}/settings/webhooks`}
-                                style="line-height: 1.5;"
-                                class="card u-width-full-line">
-                                <span class="u-flex u-cross-center u-gap-16">
-                                    <div class="avatar is-medium" aria-hidden="true">
-                                        <img
-                                            src={`${base}/icons/${$app.themeInUse}/grayscale/code.svg`}
-                                            alt="technology" />
-                                    </div>
-                                    <span class="text">Webhook</span>
-                                    <span
-                                        class="icon-plus u-margin-inline-start-auto"
-                                        style="font-size: var(--icon-size-large);"
-                                        aria-hidden="true" />
-                                </span>
-                            </a>
-                        </li>
-                    </ul>
-                </div>
-            </section>
-        {/if}
+<div class="dashboard-header">
+    <div>
+        <h1>Welcome, {$user.name}</h1>
+        <span>Follow a few quick steps to get started with Appwrite</span>
     </div>
-{/if}
-
-<article class="u-grid u-width-full-line common-section onboard-cover">
-    <img src={onBoardImage1Mobile} class="is-only-mobile u-width-full-line" alt="statistics" />
-    <img src={onBoardImage1Desktop} class="is-not-mobile u-width-full-line" alt="statistics" />
-    <div class="u-height-100-percent u-width-full-line u-padding-block-start-20">
-        <div class="u-flex u-flex-vertical u-height-100-percent u-cross-center u-padding-inline-24">
-            <div class="avatar">
-                <i class="icon-lock-closed" />
-            </div>
-            <div class="common-section u-text-center">
-                <h5 class="heading-level-5">Add a platform to view data about your project</h5>
-            </div>
-        </div>
-    </div>
-</article>
+    {#if !hasOnboardingDismissed(projectId)}
+        <Button.Button
+            variant="secondary"
+            size="s"
+            on:click={() => {
+                setHasOnboardingDismissed(projectId);
+                if (location.href.endsWith('getstarted')) {
+                    goto(`${base}/project-${projectId}`);
+                } else {
+                    location.reload();
+                }
+            }}>Dismiss this page</Button.Button>
```

Comment:
Would `Skip` or something else look better? Haven't seen the design on this one yet so.

Reply:
I think I saw this in designs, feel free to ignore and mark as resolved.